*** tests/A-10-36-M-ok.out	2018-12-03 22:36:38.000000000 +0000
--- tests/A-10-36-M-ok.outhyp	2018-12-15 02:10:26.731071824 +0000
***************
*** 20,26 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Criar inquérito: Operação inválida: O projecto 'P4' não existe na disciplina 'D2'.
  Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
--- 20,26 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Criar inquérito: Operação inválida: O projecto 'P4' da disciplina 'D2' já tem um inquérito associado.
  Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
