*** tests/A-10-11-M-ok.out	2018-12-03 22:36:38.000000000 +0000
--- tests/A-10-11-M-ok.outhyp	2018-12-17 09:52:49.573367012 +0000
***************
*** 16,21 ****
--- 16,22 ----
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
+ 6 - Remover aluno de Disciplina
  0 - Sair
  Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1
  * 100012 - Entreguei!
***************
*** 26,31 ****
--- 27,33 ----
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
+ 6 - Remover aluno de Disciplina
  0 - Sair
  Escolha uma opção: Menu principal
  1 - Abrir
