*** tests/A-10-72-M-ok.out	2018-12-05 23:19:32.000000000 +0000
--- tests/A-10-72-M-ok.outhyp	2018-12-15 02:10:34.059089509 +0000
***************
*** 5,15 ****
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projecto P2 da disciplina D1
! Pode preencher inquérito do projecto P4 da disciplina D1
! Resultados do inquérito do projecto P1 da disciplina D1
! Pode preencher inquérito do projecto P5 da disciplina D1
! Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
--- 5,11 ----
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
***************
*** 21,28 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Número de horas de trabalho: Comentário ao projecto: Preencher inquérito: Operação inválida: O projecto 'P5' não existe na disciplina 'D1'.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
--- 17,23 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Número de horas de trabalho: Comentário ao projecto: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
