*** tests/A-10-67-M-ok.out	2018-12-07 18:33:26.000000000 +0000
--- tests/A-10-67-M-ok.outhyp	2018-12-15 02:10:33.031087023 +0000
***************
*** 5,12 ****
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Resultados do inquérito do projecto P1 da disciplina D1
! Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
--- 5,11 ----
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
***************
*** 38,47 ****
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
  Escolha uma opção: Nome da disciplina: D1 - P0 (por abrir)
- D1 - P1 - 1 respostas - 1 horas
- D1 - P2 (aberto)
- D1 - P4 (aberto)
- D1 - P5 (por abrir)
  Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
--- 37,42 ----
