*** tests/A-10-41-M-ok.out	2018-12-03 22:36:38.000000000 +0000
--- tests/A-10-41-M-ok.outhyp	2018-12-15 02:10:27.815074435 +0000
***************
*** 36,42 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Cancelar inquérito: Operação inválida: O inquérito do projecto 'P1' da disciplina 'D1' já está finalizado.
  Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
--- 36,42 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Cancelar inquérito: Operação inválida: O projecto 'P1' da disciplina 'D1' não tem inquérito associado.
  Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
