*** tests/A-10-40-M-ok.out	2018-12-03 22:36:38.000000000 +0000
--- tests/A-10-40-M-ok.outhyp	2018-12-15 02:10:27.619073963 +0000
***************
*** 61,67 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
--- 61,68 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Cancelar inquérito: Operação inválida: O projecto 'P1' da disciplina 'D1' não tem inquérito associado.
! Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
***************
*** 81,87 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1 (aberto)
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
--- 82,88 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1 (fechado)
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
